sequential/utils.py

```diff
@@ -232,6 +234,10 @@ def validate_sequences(sequences: List[list]):
     check_true(len(not_list) == 0, ValueError("Sequences need to be a list of lists.", not_list))
     is_empty_list = any([len(sequences[i]) == 0 for i in range(len(sequences))])
     check_false(is_empty_list, ValueError("Sequences cannot contain any empty list."))
+    if not isinstance(sequences[0][0], str):
+        min_int = get_min_value(sequences)
+        check_true(min_int > 0, ValueError("Integers that are representing items cannot contain 0. "
+                                           "The integers must be positive integers."))
```

Comment:
Wouldn't it be sufficient to say: "Integers must be be greater than 0"?

Reply:
Re: the analysis of speed up, for example in the unit [test](https://github.com/fidelity/seq2pat/blob/batch_processing/tests/test_seq2pat_batch.py#L265) on test data with not constraints. 

-  Single thread with no batching: 1.4s
-  Single thread with batching, batch_size_10000 : 6.9s
-  8 threads with batching, batch_size_10000 : 2.1s

So there is an effect of speed up. Given a smaller set, ~60k sequences, there seems no need to go batching.